tasks/sonar_runner.js

```diff
@@ -44,7 +44,10 @@ module.exports = function (grunt) {
         var dryRun = options.dryRun;
         var done = this.async();
 
-        options.sonar.language = options.sonar.language || 'js';
+        // in case the language property isn't set sonar assumes this is a multi language project
+        if (options.sonar.language) {
+            options.sonar.language = options.sonar.language;
```

Comment:
no... forgot to delete it after I removed the "|| 'js'" fallback....

Will remove this